modules/cnv-editing-kubevirtstorageclassdefaults-cli.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-storage-defaults-for-datavolumes.adoc
+
+[id="cnv-editing-kubevirtstorageclassdefaults-cli_{context}"]
+= Editing the `kubevirt-storage-class-defaults` ConfigMap in the CLI
+
+You can edit the default storage setting by modifying the `kubevirt-storage-class-defaults` ConfigMap in the `openshift-cnv` namespace.
+
+If you want to create DataVolumes using different storage settings, you can add storage settings for storage classes in the ConfigMap.
+It is important that you configure storage settings that are supported by the underlying storage.
+
+.Procedure
+
+. Use `oc edit` to edit the Config Map:
++
+----
+$ oc edit configmap kubevirt-storage-class-defaults -n openshift-cnv
+----
+
+. Update the `data` values of the Config Map:
++
+[source,yaml]
+----
+...
+data:
+  accessMode: ReadWriteOnce <1>
+  volumeMode: Filesystem <2>
+  <new>.accessMode: ReadWriteMany <3>
+  <new>.volumeMode: Block <4>
+----
+<1> The default accessMode is `ReadWriteOnce`.
+<2> The default volumeMode is `Filesystem`. 
+<3> If you add a settings for a storage class, replace the `<new>` part of the parameter with the storage class name.
+<4> If you add a settings for a storage class, replace the `<new>` part of the parameter with the storage class name.
```

Comment:
3 and 4: should these both say "a setting"? I'd also maybe phrase it like "You can add a setting for a storage class by replacing `<new>` with the storage class name."

Reply:
It's entirely hypothetical as to whether the user adds those lines into the configmap so an 'if' statement is required. 
However I updated those two lines so that it is 'If you add an access mode for ...' and 'If you add a volume mode for...'